invenio_app_ils/config.py

```diff
@@ -603,6 +613,51 @@ def _(x):
         update_permission_factory_imp=record_update_permission_factory,
         delete_permission_factory_imp=backoffice_permission,
     ),
+    dreqid=dict(
+        pid_type=DOCUMENT_REQUEST_PID_TYPE,
+        pid_minter=DOCUMENT_REQUEST_PID_MINTER,
+        pid_fetcher=DOCUMENT_REQUEST_PID_FETCHER,
+        search_class=DocumentRequestSearch,
+        record_class=DocumentRequest,
+        indexer_class=DocumentRequestIndexer,
+        search_factory_imp="invenio_app_ils.search.api"
+                           ":filter_by_patron_search_factory",
+        record_loaders={
+            "application/json": (
+                "invenio_app_ils.records.loaders:document_request_loader"
+            ),
+            "application/json-patch+json": (
+                lambda: request.get_json(force=True)
+            ),
+        },
+        record_serializers={
+            "application/json": (
+                "invenio_app_ils.records.serializers:json_v1_response"
+            ),
+        },
+        search_serializers={
+            "application/json": (
+                "invenio_records_rest.serializers:json_v1_search"
+            ),
+            "text/csv": ("invenio_app_ils.records.serializers:csv_v1_search"),
```

Comment:
If we add this `csv`, I would add also the `Export` button then to the Search page! :)

Reply:
It's already there and I tested it and works great! :)